Makefile

```diff
@@ -7,8 +7,8 @@
 
 BUILD_DIR = OBJROOT="$(CURDIR)/build" SYMROOT="$(CURDIR)/build"
 SHELL = /bin/bash -e -o pipefail
-IOS32 = -scheme OCMockLib -sdk iphonesimulator -destination 'name=iPhone 5' $(BUILD_DIR)
-IOS64 = -scheme OCMockLib -sdk iphonesimulator -destination 'name=iPhone 5s' $(BUILD_DIR)
+IOS32 = -scheme OCMockLib -destination 'platform=iOS Simulator,OS=10.3.1,name=iPhone 5' $(BUILD_DIR)
+IOS64 = -scheme OCMockLib -destination 'platform=iOS Simulator,OS=latest,name=iPhone X' $(BUILD_DIR)
```

Comment:
A little bit of future proofing. Judging by the age of the device, it may get dropped by Apple soon, so when migrating Xcode versions in the future, it'll be one less thing to worry about. Also, when Apple finally adds arm64e to the default build toolchain, a parallel destination can be added such as the Xs or Xr with very little code change.

Reply:
I see. Makes sense.